docs/understanding-tracking-design/introduction-to-tracking-design/best-practices/index.md

```diff
@@ -0,0 +1,248 @@
+---
+title: "Tracking design best practices"
+---
+
+Designing the events and entities to be captured in your tracking is a challenging task.
+This page aims to give guidance on how to approach it based on our experience.
+
+Overall, it suggests taking the following steps:
+
+1. Analyze the business use case and define the reports to be produced.
+2. Check our out-of-the-box data products to see if you can reuse or build on top of them.
+3. Choose the naming convention to follow in your schemas.
+4. Define your entities first.
+5. Introduce events as interactions of the entities.
+
+## Analyze your business use case and define reports
+
+Good tracking design comes from an understanding of the business use case.
+Identifying and analysing the use case should be the first step before we design our tracking.
```

Comment:
There's a bit of a mix of american/british going on thoughout the doc, for example here you have used `analysing`, whereas the heading uses `Analyze`.